builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,179 @@
+
+    ```bash
+    pnpm add moralis
+    ```
+
+=== "pip"
+
+    ```bash
+    pip install moralis
+    ```
+
+Once the SDK is ready, you can leverage it to query Moralis APIs. Here's an example of how to interact with the EVM API using a JSON-RPC interface:
+
+=== "JavaScript"
+
+    ```js
+    import Moralis from 'moralis';
+
+    try {
+      await Moralis.start({
+        apiKey: 'INSERT_YOUR_API_KEY',
+      });
+     
+      const response = await Moralis.EvmApi.block.getBlock({
+        chain: '0x504',
+        blockNumberOrHash: '1',
+      });
+      
+      console.log(response.raw);
+    } catch (e) {
+      console.error(e);
+    }
+    ```
+
+=== "Python"
+
+    ```python
+    from moralis import evm_api
+
+    api_key = "INSERT_YOUR_API_KEY"
+
+    params = {"chain": "moonbeam", "block_number_or_hash": "1"}
+
+    result = evm_api.block.get_block(
+        api_key=api_key,
+        params=params,
+    )
+
+    print(result)
+    ```
+
+For more information on advanced features and configurations within the Moralis SDK, please refer to the official [JavaScript](https://moralisweb3.github.io/Moralis-JS-SDK/Introduction){target=\_blank} and [Python](https://moralisweb3.github.io/Moralis-Python-SDK/){target=\_blank} documentation.
+
+## Accessing the Stream API {: #accessing-stream-api }
+
+The Moralis Streams API is a tool for developers to listen to and react to events happening on blockchains in real time. This API lets you listen for specific events, such as a new transaction being added to a block, a particular smart contract function being called, or an NFT being transferred and delivering the event via Webhook.
+
+Head over to the **Streams** page from your Moralis dashboard and click **Create a new stream** to get started.
+
+![stream API page](/images/builders/integrations/indexers/moralis/moralis-2.webp)
+
+To get started quickly, choose a predefined template. This example focuses on transactions related to xcDOT. Select the **Contract Activity** template to listen for events related specifically to xcDOT.
+
+![template selection page](/images/builders/integrations/indexers/moralis/moralis-3.webp)
+
+On the next page, enter the following details:
+
+1. Enter the smart contract address for [xcDOT](https://moonscan.io/token/0xffffffff1fcacbd218edc0eba20fc2308c778080){target=\_blank}: 0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080
+2. Select Moonbeam from the available network options. Moralis supports listening to events on Moonbeam, Moonriver, and Moonbase Alpha 
+3. Customize the events you want to receive. For this example, choose **All contract events**. You may also test the stream to confirm it captures the needed data
+4. Connect the stream to an endpoint to receive updates from Moralis
+5. Click **Save** to activate your stream
+
+![details input page](/images/builders/integrations/indexers/moralis/moralis-4.webp)
+
+All set. Once you activate your stream, it will be added to your list of streams on your dashboard:
```

Comment:
Try to avoid sentence fragments like All set. Now you're all set or something like that would be fine.